### DISCUSSION TITLE
Default remote user group not working

### DISCUSSION THREAD

**Discussion by PaulR282** [netbox-community/netbox#14489] [Help Wanted!]
We run netbox for some while with LDAP configuration and everything is working so far. Now I wanted to add a default group that every user should get. I created the group and set `REMOTE_AUTH_DEFAULT_GROUPS = ['user']` in the `configuration.py`. Then I tested it by deleting a user, that was created by the LDAP connection and logged back in with that account. The user was re-created but no group was assigned. There is nothing shown in the log apart from the debug information about the user creation.

**Comment by candlerb** [comment]
Issue #8854 said that this didn't work for social auth, and there was a fix specific for that at #9600. Maybe a similar fix is required for LDAP?

There a function `configure_user()` in netbox/netbox/authentication.py which looks at `REMOTE_AUTH_DEFAULT_GROUPS`, but it appears to apply only to RemoteUserBackend (which is when the HTTP proxy provides headers identifying the user)

**Reply by PaulR282** [reply]
Ok, so this does not work. Do you know a way to assign a default group to any user automatically besides a custom script?

**Reply by PaulR282** [reply]
I also tried this https://www.reddit.com/r/Netbox/comments/u8ybmm/automatically_add_users_from_ldap_group_to_custom/ but it didn't work (the groups were not added)

**Reply by candlerb** [reply]
Sorry I don't have a solution for you. All I'm observing is that Netbox features relating to "remote auth" may not necessarily apply to LDAP and/or Social Auth, since they are completely different auth backends and "Remote Auth" refers to HTTP proxy auth.
